opimodel/widgets.py

```diff
@@ -221,6 +222,7 @@ def __init__(self, x, y, width, height, pv):
             TextInput.TYPE_ID, x, y, width, height)
 
         self.pv_name = pv
+        self.horizontal_alignment = HAlign.CENTER
```

Comment:
Is this necessarily better than whatever CS-Studio inserts as the default alignment?